.github/workflows/automationTemplate.yaml

```diff
@@ -51,10 +44,16 @@ jobs:
             releaseDate=`date +%F`
             echo "Defaulting to todays date ${releaseDate}"
           fi
+          echo "RELEASE_DATE=${releaseDate}" >> $GITHUB_ENV
 
-          echo "releaseDate=${releaseDate}" >> $GITHUB_ENV
+          declare -A platformsCamelCase
+          echo "platformName: ${{ inputs.platformName }}"
+          echo "platformNameCamelCase: ${platformNamesCamelCase[${{ inputs.platformName }}]}"
+          platformNamesCamelCase=( ["dotnet"]="DotNetCore" )
```

Comment:
I'm wondering if we should make the platform name even more descriptive. Here, I don't believe that anything indicates to us that the `DotNetCore` name must actually correspond to what we use in the base pipeline, and so someone could expect `dotnet` to be `DotNet`.
Please consider renaming to something like `platformNamesAsInBasePipeline` or something like that.

Reply:
Agreed & updated.